src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/notifications/ReferralNotificationService.kt

```diff
@@ -109,6 +128,26 @@ class ReferralNotificationService(
     }
   }
 
+  private fun sendEmail(
```

Comment:
Since this email is specific to `completionDeadlineUpdatedTemplateID`, what do you think about using a specific function name?

Reply:
Agreed @sldblog , I will make those changes